### PR TITLE
Add docker image to be a manifest list

### PR DESCRIPTION
**Pull request by rajansandeep**
### 1. What does this pull request do?
Includes Docker image to be a manifest list.

### 2. Which issues (if any) are related?
#1177

### CODE REVIEW

File: Makefile.release [hunk 1]
```diff
@@ -38,6 +44,10 @@ ifeq (, $(shell which gh-release))
      $(error "No gh-release in $$PATH, install with `go get github.com/progrium/gh-release`")
 endif
 
+ifeq (, $(shell ls | grep manifest-tool ))
```
Comment (miekg) [inline Makefile.release]: Don't assume this is locally installed. Might be in $PATH.

**Review by miekg — CHANGES_REQUESTED** [review]

**Review by miekg — APPROVED** [review]

**Comment by yongtang** [comment]
@rajansandeep The base image has to be from different architecture and the binary should used the generated binary form different directories. It should not use the same binary.

**Comment by yongtang** [comment]
The binary seems to be fine as `tar -xzf ` is used. Need to change the base image.

**Comment by miekg** [comment]
@yongtang what remains to be done here?
(If we want to do a quick 0.9.10, this the last outstanding issue)

**Comment by rajansandeep** [comment]
Per the docs: https://hub.docker.com/r/library/alpine/ 

Supported Architectures:
amd64, arm32v6, arm64v8, i386, ppc64le, s390x
As of 2017-09-12, these other architectures are included under the non-prefixed images via "manifest lists" .

So we should be fine as is.

**Comment by yongtang** [comment]
@rajansandeep @miekg Let's hold on for that. The base image is multi arch. However, at the time the coredns docker image is built, the `docker build` is running on a amd64 machine (the dev machine) and the binary will only be added to amd64 docker image.

There are some additional steps to be done. See steps in:
https://github.com/luxas/kubeadm-workshop/blob/master/images/nodejs/Makefile

I am preparing the PR and will open the PR shortly.

**Comment by miekg** [comment]
ok

**Pull request merged**